scanner/matcher/matcher.go

```diff
@@ -40,43 +42,68 @@ func NewMatcher(ctx context.Context, cfg config.MatcherConfig) (Matcher, error)
 	if err != nil {
 		return nil, fmt.Errorf("initializing postgres matcher store: %w", err)
 	}
+	metadataStore, err := metadatapostgres.InitPostgresMetadataStore(ctx, pool, true)
+	if err != nil {
+		return nil, fmt.Errorf("initializing postgres matcher metadata store: %w", err)
+	}
 	locker, err := ctxlock.New(ctx, pool)
 	if err != nil {
 		return nil, fmt.Errorf("creating matcher postgres locker: %w", err)
 	}
-	// TODO: Remove when Scanner V4 updater pipeline is available.
-	if err := updaterdefaults.Error(); err != nil {
-		return nil, fmt.Errorf("vulnerability updater init: %w", err)
-	}
+
 	// TODO: Update HTTP client.
 	c := http.DefaultClient
+
 	libVuln, err := libvuln.New(ctx, &libvuln.Options{
```

Comment:
Maybe we want to wait for at least one successful updater run before we say "libvuln is ready". That would help with the initialization part. Fine to leave this outside the PR.